leakcheck.supp

```diff
@@ -1,3 +1,10 @@
+{  
```

Comment:
Why is this needed?

Reply:
It pops up in macOS. Just to clear noise. It's not essential, I can remove it.

Reply:
But maybe its not noise?

Reply:
That's possible. I can reproduce it and we'll look into it.